strippy.ps1

```diff
@@ -245,6 +248,21 @@ function log ([String] $Stage, [LEnum] $Type = [LEnum]::Trace, [String] $String,
     }
 }
 
+# Help flag checks
+if ($h -match "true" -or $help -match "true" ) {
+    log params trace "Strippy was started help flags. Showing the get-help output for the script and exiting"
+    Get-Help $(join-path $(Get-Location) $MyInvocation.MyCommand.Name)
```

Comment:
We should probably standardise the help outputs, so this line will need a `-Detailed` appended